prow/github/client.go

```diff
@@ -419,7 +419,7 @@ func (c *Client) IsMember(org, user string) (bool, error) {
 
 // CreateComment creates a comment on the issue.
 func (c *Client) CreateComment(org, repo string, number int, comment string) error {
-	c.log("CreateComment", org, repo, number, comment)
+	c.log("CreateComment", org, repo, number)
```

Comment:
I know that in some cases I was using the content of this comment field to determine _what_ `prow` was doing -- if we bump this to `Info` but lose the content do we still have a way to determine what the action is? Might need to add a "why" log in the plugins themselves.

Reply:
Added back, not terrible to have.